examples/smpi/replay_multiple_manual_deploy/replay_multiple_manual.cpp

```diff
@@ -232,12 +218,12 @@ static std::vector<Job*> all_jobs(const std::string& workload_file)
           throw std::runtime_error("Cannot open file " + job->filename);
 
         std::string traces_line;
-        while (getline(traces_file, traces_line)) {
+        while (std::getline(traces_file, traces_line)) {
           boost::trim_right(traces_line);
           job->traces_filenames.push_back(dir + "/" + traces_line);
         }
 
-        if ((int)job->traces_filenames.size() < job->app_size)
+        if (static_cast<int>(job->traces_filenames.size()) < job->app_size)
```

Comment:
app_size should probably be an unsigned int; this cast would then be unnecessary